### DISCUSSION TITLE
mkdocs-material/blog/2023/11/30/adding-a-badge-to-your-project/

### DISCUSSION THREAD

**Discussion by giscus** [squidfunk/mkdocs-material#6412] [_]
# mkdocs-material/blog/2023/11/30/adding-a-badge-to-your-project/

Share the love – you can now add a badge to your README, showing that your project is built with Material for MkDocs

https://squidfunk.github.io/mkdocs-material/blog/2023/11/30/adding-a-badge-to-your-project/

**Comment by cirospat** [comment]
It's a simple and easy way to give credit to **MATERIAL FOR MKDOCS** in every documentation project

**Reply by cirospat** [reply]
I also put the badge
[![Built with Material for MkDocs](https://img.shields.io/badge/Material_for_MkDocs-526CFE?style=for-the-badge&logo=MaterialForMkDocs&logoColor=white)](https://squidfunk.github.io/mkdocs-material/) inside the documentation project pages:
https://opendatasicilia.github.io/guida-umap/informazioni/

**Comment by alexvoss** [comment]
Now what we need is Material swag! I want the mug, the t-shirt and the hoodie!

**Comment by obscurefreeman** [comment]
[![Built with Material for MkDocs](https://img.shields.io/badge/Material_for_MkDocs-526CFE?style=for-the-badge&logo=MaterialForMkDocs&logoColor=white)](https://squidfunk.github.io/mkdocs-material/)
I love it !!!

**Comment by mikigo** [comment]
I have pasted it into my project, cool ~~

https://linuxdeepin.github.io/deepin-autotest-framework/

**Reply by mikigo** [reply]
[![Built with Material for MkDocs](https://img.shields.io/badge/Material_for_MkDocs-526CFE?style=flat&logo=MaterialForMkDocs&logoColor=white&color=%23F79431)](https://squidfunk.github.io/mkdocs-material/)

**Comment by LeeYeungYeung** [comment]
[![Built with Material for MkDocs](https://img.shields.io/badge/Material_for_MkDocs-526CFE?style=for-the-badge&logo=MaterialForMkDocs&logoColor=white)](https://squidfunk.github.io/mkdocs-material/)

**Comment by w1ndys** [comment]
[![Built with Material for MkDocs](https://img.shields.io/badge/Material_for_MkDocs-526CFE?style=for-the-badge&logo=MaterialForMkDocs&logoColor=white)](https://squidfunk.github.io/mkdocs-material/)

so  cool

**Comment by electronick-co** [comment]
nice!
I added it to my project https://www.deepdeck.co

**Comment by Artefactslimited** [comment]
Ok

**Comment by picli3** [comment]
Done!!

**Comment by uzvg** [comment]
[![Built with Material for MkDocs](https://img.shields.io/badge/Material_for_MkDocs-526CFE?style=for-the-badge&logo=MaterialForMkDocs&logoColor=white)](https://squidfunk.github.io/mkdocs-material/)

It's really cool!!
